packages/services/src/services/Locale/Locale.js

```diff
@@ -144,14 +144,14 @@ async function _getLocaleFromDDO() {
       }
     }
 
-    if (!pageInfoIBM.lc || !pageInfoIBM.cc) return false;
+    if (!pageInfoIBM.lc || !pageInfoIBM.cc) return _getLocaleByLangAttr();
```

Comment:
@reesedownes looking at the logic in getLang, I wonder if it would be ok to leave it as-is with returning `false`, just so we can keep the logic of fetching from DDO and fetching from lang separate.

Reply:
@jeffchew yeah, that's pretty heavy handed. i'll try it that way quickly.